generatePackages.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/sh
+
+cp README.md ./build/
+rm -rf ./build/types/internal
```

Comment:
Add a `mkdir -p ./build` just in case `build` dir is removed.

Reply:
done